src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import core.basesyntax.model.AreaCalculator;
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Figure;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class HelloWorld {
+    public static void main(String[] args) {
+
+        System.out.print(String.format("%s. Color: %s. ", this.getClass().getSimpleName(), color));
+        final int amountFigures = 6;
+        final int HALF_FIGURES_COUNT = 3;
+        AreaCalculator figure = new Circle();
```

Comment:
The `AreaCalculator` interface cannot be instantiated directly. You should instantiate a class that implements this interface, such as `Circle`, and ensure it has the necessary methods implemented.